addons/website_slides/static/lib/pdfslidesviewer/PDFSlidesViewer.js

```diff
@@ -148,6 +148,23 @@ var PDFSlidesViewer = (function(){
     PDFSlidesViewer.prototype.toggleFullScreen = function(){
         var el = this.canvas;
 
+        var $PDFViewer = $("div#PDFSlideViewer")
+        var $navigation_footer = $('div#PDFViewer > div.navbar-fixed-bottom')
+
+        $(document).unbind('webkitfullscreenchange mozfullscreenchange fullscreenchange')
```

Comment:
Because every-time we go full-screen, toggleFullScreen will be called and fullscreenchange events will get bound to dom. For example if we are going full-screen second time, those events will be bounded twice and clicking fullscreen-ed element will toggle the footer twice. So clearing the event stack first and than binding events.